src/main/java/com/google/api/gax/grpc/BundlingCallSettings.java

```diff
@@ -68,6 +68,12 @@ public Builder(MethodDescriptor<RequestT, ResponseT> grpcMethodDescriptor,
       this.bundlingDescriptor = bundlingDescriptor;
```

Comment:
I just noticed - bundlingSettingsBuilder isn't set by default, but I think it should be. Also I think we should ensure that the set method doesn't set it to null.